xLights/wxsmith/RipplePanel.wxs

```diff
@@ -45,12 +98,38 @@
 					<size>14,14</size>
 					<bg>wxSYS_COLOUR_BTNHIGHLIGHT</bg>
 					<style>wxBU_AUTODRAW|wxBORDER_NONE</style>
```

Comment:
I cannot get wxSmith to stop clobbering the action handlers.  In the end, I let it win, and resorted to adding the connect call manually, see the .cpp file where they are carefully moved outside the generated section.